server/game/cards/13.4-BtRK/ShadowblackAssassin.js

```diff
@@ -0,0 +1,24 @@
+const DrawCard = require('../../drawcard');
+
+class ShadowblackAssassin extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onCardOutOfShadows: event => event.card === this
+            },
+            target: {
+                type: 'select',
+                activePromptTitle: 'Select a character with STR 1',
+                cardCondition: card => card.location === 'play area' && card.controller !== this.controller && card.getType() === 'character' && card.getStrength() === 1

```

Comment:
The card text says "STR 1 or lower", so your comparison should be <= 1. Additionally, you should be able to kill your own characters, so no need for the controller check.